firebase/test/collections/integrity-check.spec.ts

```diff
@@ -1,6 +1,61 @@
+import * as assert from "assert";
+import * as fs from "fs";
+import Provider from "#presentation/provider";
+
 describe("Collections Integrity Check", () => {
-  // TODO(matuella):
-  // TEST 1: Maintain naming + id consistency
-  // TEST 2: No duplicate memos ids amongst each collection
-  // TEST 3 (?): should we check for "data consistency", such as at least 1 contributor, req. description, req. locale, etc.
+  let localCollectionsPaths: string[];
+  let rawLocalCollections: any[];
+
+  before(async () => {
+    localCollectionsPaths = await fs.promises.readdir(`${process.cwd()}/collections`);
+
+    const fsGateway = Provider.instance.fileSystemGateway;
+    const localCollections = await fsGateway.readDirFilesAsStrings("./collections");
+    rawLocalCollections = localCollections.map((collection) => JSON.parse(collection));
+  });
+
+  it("should have at least one collection", () => assert.ok(rawLocalCollections.length > 0));
+
+  it("should maintain naming/id consistency", () => {
+    for (let index = 0; index < localCollectionsPaths.length; index++) {
+      const collectionId: string = rawLocalCollections[index]["id"];
+      const fullCollectionPath = localCollectionsPaths[index]!;
+      assert.ok(
+        fullCollectionPath.includes(`${collectionId}.json`),
+        `Collection had id "${collectionId}" but file path "${fullCollectionPath}"`
+      );
+    }
+  });
+
+  it("should have unique ids amongst themselves", () => {
+    const collectionIds: string[] = [];
+    rawLocalCollections.forEach((collection) => {
+      const collectionId: string = collection["id"];
+      assert.ok(!collectionIds.includes(collectionId), `Duplicate collection id "${collectionId}"`);
+      collectionIds.push(collectionId);
+    });
+  });
+
+  it("should have unique memo ids amongst themselves", () => {
```

Comment:
```suggestion
  it("should have unique memo ids in each collection", () => {
```

Why should they have a unique global id?

Reply:
I copied this test from the client, I though it was a definition, but we may remove this test